src/System.IdentityModel.Tokens.Jwt/JwtSecurityToken.cs

```diff
@@ -330,6 +375,20 @@ internal set
         /// </summary>
         public JwtSecurityToken InnerToken { get; internal set; }
 
+        /// <summary>
+        /// Gets the 'value' of the 'kid' claim { kid, 'value' } from the <see cref="JwtHeader"/>.  
+        /// </summary>
+        /// <remarks>If the 'kid' claim is not found, an empty string is returned.</remarks>
+        public string Kid
+        {
+            get
+            { 
+                if (Header != null)
+                    return Header.Kid;
+                return String.Empty;
```

Comment:
It is recommended to use lowercase 'string'. 
We normally see a empty line after an 'if' clause.

Reply:
I've changed it to a lowercase 'string' here and in all other places for consistency.

I've also added empty lines after all the 'if' statements in this class.